reference/tax_zone_check.v3.yml

```
@@ -0,0 +1,184 @@
+openapi: '3.0.0'
+info:
+  title: Tax Zone Check
+  description: Check what tax zone a given address and customer group derives to.
+  termsOfService: 'https://www.bigcommerce.com/terms'
+  contact:
+    name: BigCommerce
+    url: 'https://www.bigcommerce.com'
+    email: support@bigcommerce.com
+  version: ''
+security:
+  - X-Auth-Token: [ ]
+tags:
+  - name: Tax Zone Check
+servers:
+  - url: 'https://api.bigcommerce.com/stores/{store_hash}/v3'
+    variables:
+      store_hash:
+        default: store_hash
+        description: Permanent ID of the BigCommerce store.
+    description: BigCommerce API Gateway
+paths:
+  "/tax/zonecheck":
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+    post:
+      tags:
+        - Tax Zone Check
+      summary: Check zone given an address.
+      description: Check what tax zone a given address and customer group derives to.
+      operationId: zone-check
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/ZoneCheck'
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/TaxZone'
+        "422":
+          description: The request body does not meet specifications.
+components:
+  parameters:
+    Accept:
+      name: Accept
+      in: header
+      required: true
+      description: 'The [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) of the response body.'
+      schema:
+        type: string
+        default: 'application/json'
+    ContentType:
+      name: Content-Type
+      in: header
+      required: true
+      description: 'The [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) of the request body.'
+      schema:
+        type: string
+        default: 'application/json'
+  schemas:
+    ZoneCheck:
+      type: object
+      properties:
+        country_code:
+          type: string
+          description: Two-letter ISO 3166-1 country code.
+          example: 'AU'
+        subdivision_codes:
+          type: string
+          description: Three-letter ISO 3166-2 subdivision code.
+          example: 'NSW'
+        postal_code:
+          type: string
+          description: Postal Code.
```

Worth any additional guidance here? But also:
```suggestion
          description: Postal code.
```